stl/inc/type_traits

```diff
@@ -563,6 +565,7 @@ struct is_standard_layout : bool_constant<__is_standard_layout(_Ty)> {}; // dete
 template <class _Ty>
 _INLINE_VAR constexpr bool is_standard_layout_v = __is_standard_layout(_Ty);
 
+#if !_HAS_CXX20

```

Comment:
Why is `is_literal_type` controlled by `_HAS_CXX20` instead of the `_HAS_DEPRECATED_TYPETRAITS` escape hatch?

Reply:
I think i missed it. Fixed